### ISSUE TITLE
Update Teams workshop ppt

### ISSUE THREAD

**Issue by downtownmtb** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#2650]
When will this information be updated? Some of the ppt's and docs are getting dated and missing key information on the changes in Teams in the past year. Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 377ed460-94ef-6528-63de-87360156b71e
* Version Independent ID: 56e32c63-dadb-fb81-c04e-6cf396963f2d
* Content: [Tools and workshops - Plan, deliver, and adopt Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/planning-workshop-practical-guide#feedback)
* Content Source: [Teams/planning-workshop-practical-guide.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/planning-workshop-practical-guide.md)
* Service: **msteams**
* GitHub Login: @LolaJacobsen
* Microsoft Alias: **lolaj**

**Comment by scanum** [comment]
@downtownmtb Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by lindspea** [comment]
@downtownmtb Thank you for reaching out and helping improve MS Docs.  Is there anything specific that you are referring to?  If so, please could you send me the link that will be helpful to users.  If you think this link will be helpful, I can have it added to the article - https://support.office.com/article/microsoft-teams-video-training-4f108e54-240b-4351-8084-b1089f0d21d7.  Please let me know.

**Comment by downtownmtb** [comment]
Hey! I just noticed that the sections on tenant-wide settings and config options could use a refresh since much has changed since December 2017, especially around voice options. Also, I don't see updated info for planning a migration from on-prem or online SfB in there. Perhaps that is in another workshop resource, but it should be mentioned. Those are just a couple things that I can think of, but it's hard to keep track of all the changes over the past 18 months and try to include them in this post. Thanks!

**Comment by lindspea** [comment]
@downtownmtb Thank you for your guidance.  Here are two links I thought would be helpful to add to the article.  Let me know what you think.   https://docs.microsoft.com/skypeforbusiness/hybrid/move-users-from-on-premises-to-teams and https://docs.microsoft.com/microsoftteams/enable-features-office-365.

**Comment by downtownmtb** [comment]
Yes! Those would be great resources to refer to. I found this partner resource for SfB on-prem to Teams planning workshop that may also be helpful, especially for partners. https://www.microsoft.com/microsoft-365/partners/intelligentcommunications/skype-for-business-teams

**Comment by scanum** [comment]
@downtownmtb Thank you for submitting feedback. We understand that this issue has been resolved.
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.
